gradle/android_common.gradle

```diff
@@ -92,7 +92,7 @@ android {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$gradle.kotlin_version!!"
```

Comment:
What happens if you just remove the whole line?

Reply:
Unfortunately no. It seems one of the android dependencies uses the 1.7.10 std lib. This is problematic because `TimeMark` has been changed from an abstract class to an interface and that causes the tests for date-time to not compile on Android.

Reply:
It's better to exclude the sub dependency from the dependency as I think this will force all consumers on this stdlib.